packages/utils/src/period/period.js

```diff
@@ -125,12 +125,18 @@ const checkNonNumericPeriod = (period, potentialType) => {
   return isValidFormat;
 };
 
+export const isValidPeriod = period => typeof period === 'string' && !!periodToType(period);
+
 /**
- *
- * @param {any} period
- * @returns {boolean}
+ * @returns {number} > 0 if periodA > periodB, 0 if periodA = periodB, < 0 if periodA < periodB
  */
-export const isValidPeriod = period => typeof period === 'string' && !!periodToType(period);
+export const comparePeriods = (periodA, periodB) => {
```

Comment:
I believe there would be lots of places where we could use that, this justifies the attempt for rigourous test cases. We also don't want to risk nasty bugs because of wrong comparisons (eg being different period types!)

But I won't refactor out of scope code in this PR

Reply:
Makes sense!